src/runway/config.py

```diff
@@ -536,7 +532,8 @@ def load_from_file(cls, config_path):
             config_file = yaml.safe_load(data_file)
             result = Config(config_file.pop('deployments'),
                             config_file.pop('tests', []),
-                            config_file.pop('ignore_git_branch', False))
+                            (config_file.pop('ignore-git-branch', False) or
+                             config_file.pop('ignore_git_branch', False)))
```

Comment:
Should we use a consistent method for handing `-/_`?

```suggestion
                            config_file.pop('ignore_git_branch',
                                            config_file.pop('ignore-git-branch',
                                                            False))
```